index.css

```diff
@@ -76,3 +74,162 @@ body {
 .social-icons li {
   list-style-type: none;
 }
+
+/* ----------Works page styling begins here----- */
+
+.works-card {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  margin: 8em 7% 0 7%;
+  padding: 1em 1em 2em 1em;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  color: #172b4d;
+  background-color: fff;
+}
+
+.works-title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.first_buttons {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  padding-top: 10px;
+  justify-content: flex-start;
+}
+
+.first_buttons a {
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 8px;
+  text-decoration: none;
+  padding: 4px 12px;
+}
+
+.card_details p {
+  width: 95%;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.card_details ul {
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  font-weight: 600;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 20px;
+}
+
+.canopy {
+  color: #344563;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  list-style: none;
+}
+
+.works-text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.last-button {
+  display: flex;
+  flex-direction: row;
+  padding-top: 24px;
+  padding-bottom: 16px;
+}
+
+.first_buttons,
+.coding-languages ul {
+  list-style: none;
+}
+
+/* -------------- About page styling begins here -------- */
+
+.about_page {
+  height: 100vh;
+}
+
+.works-card-2 {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] This section is not similar to the Figma design. Kindly consider exporting the background image of about section in Figma design and apply here in about section as `background-image`.
- [ ] Also consider applying button styling changes here as I mentioned  in the `see project` button.
you design
![asd](https://user-images.githubusercontent.com/96683843/212402971-49960770-8f98-4cd6-b8b3-70c88b926255.jpg)

figma design
![asdf](https://user-images.githubusercontent.com/96683843/212403034-d9af5ecd-d7db-4ccb-a153-3650f7c9ed3f.jpg)